tce/Python-bindings/scripts/cfgTest.py

```diff
@@ -39,34 +39,34 @@ def printOut(pr):
         immediateSum = immediateSum + stats.immediateCount()
         instructionSum = instructionSum + stats.instructionCount()
         bypassSum = bypassSum + stats.bypassedCount()
-        blocks = blocks + stats.normalBBCount()        
-    print "Program:"+sys.argv[1]
-    print "\tMoves:",moveSum  
-    print "\tImmediates:",immediateSum 
-    print "\tInstructions:",instructionSum 
-    print "\tBasicBlocks:",blocks
-    print "\tProcedures:",pr.procedureCount()
-    print "\tMovesPerCycle:",1.0*moveSum/instructionSum 
+        blocks = blocks + stats.normalBBCount()
+    print("Program:"+sys.argv[1])
```

Comment:
The style looks tight. Can you add white spaces around the + operator?

Reply:
Okay if you are really want and care about git blame over these improvements I can make it. I guess if you follow this mentality then upgrading code formatting cannot basically be done? I can remove the changes of course if that's what is needed :) I personally care about avoiding unnecessary white spaces and follow PEP8 formatting. And usually fix some on the same way when improving and fixing the code base. Leaving things "cleaner" than before on the process. Mostly those edits are coming from my used formatters from my editor. Of course the code is full of format errors and this change just touches the surface.

Reply:
Overall, maintaining easy 'git blame' is extremely valueable in a long running project like TCE with a lot of developers that come and go. But in this particular case I don't see so much value so I'll just pull this in. But in the future, style updates are preferably done only nearby the actual changed logic and in a separate commit to make log readability better (bunch of style updates can bury the interesting changes in the noise).